prow/plugins/size/pr.go

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Put the contents of both this file and "plugin.go" into one file called "size.go".